templates/rcac/modules/tcl/default.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# This script prints a TCL module file for a Python environment.
+# All the variables and conditional statements are evaluated inside "conda-env-mod"
+# using the "source" function in bash.
+# -----------------------------------------------------------------------------------------
+# Useful varibales defined by "conda-env-mod"
+# ENV_DIR := path to environment
+# $PY := full pyhton version, e.g., 3.10.11
+# $PY_S := short python version, e.g., 3.10
+# Opt_LOCAL_PY := 0|1, 
+#						1 means use python from the environment
+#						0 means use python from base anaconda
+#------------------------------------------------------------------------------------------
+
+cat <<-EOF
+	#%Module
+	# created on $(date) by $USER
+
+EOF
+
+if [ "${CONDA_MODULE}" != "" ]; then
+	cat <<-EOF
+		module load "${CONDA_MODULE}"
+	EOF
+elif [ -f "${CONDA_PREFIX}/bin/conda" ]; then
+	cat <<-EOF
+		# Use the base conda that created this environment.
+		prepend-path PATH "${CONDA_PREFIX}/bin"
+	EOF
+else
+	echo -e "WARNING: Couldn't find an anaconda module.\nPlease make sure you initialize conda separately." >&2
+fi
+
+cat <<-EOF
+	set modroot "${ENV_DIR}"
+	set pyver "${PY}"
+
+	set-alias pip "\$modroot/bin/pip"
+	# If this is Python3 environment, brace against 'pip3', too
+	if {[regexp "^3\." "$pyver" match]} {
+	    set-alias pip3 "\$modroot/bin/pip"
+	}
```

Comment:
Do we still want this? Especially with the new default of--local-python, I do not see the benefit of aliasing pip, and it would be unexpected and confusing for users.

I think these aliases are only needed for --base-python scenario, so probably need to be wrapped into a conditional.